### DISCUSSION TITLE
404 error with api calling

### DISCUSSION THREAD

**Discussion by lexthedev** [Budibase/budibase#8842] [Help]
Hello, developers!

I am trying to "touch" your product in hope to use it (even may be with pro package), but have an issue with starting the app.
I have these software versions:

- Debian: 11 (also have tried with Win 10 + WSL 2 and Ubuntu);
- Docker: 20.10.5;
- Docker compose: 2.13.0;
- Node: 14.20.1;
- Lerna: 5.5.4.

And have these errors:

![image](https://user-images.githubusercontent.com/44236172/204303986-ff97d918-316c-4ebf-b13b-cd8b1c7585e4.png)

![image](https://user-images.githubusercontent.com/44236172/204303705-bfd1621a-0026-40b8-bf0b-e7ac8e2c56ca.png)

![image](https://user-images.githubusercontent.com/44236172/204303611-49743176-283a-431c-aecc-23f7bc40d49d.png)

ports are 3000 and 10000, but situations are the same.

Looking forward for your response! 
Have a good day )) 👍

**Comment by Kirill-code** [comment]
Wow, I have same issue. Thought that it is only my fault...

**Comment by melohagan** [comment]
Hey @lexthedev 

So you are port forwarding to 3000. You have tried the default 10000 settings and same issue.

What method of installation are you using?
https://docs.budibase.com/docs/hosting-methods

The Budibase CLI is probably the easiest way to self host, but you can also setup a cloud account here: https://account.budibase.app/register

**Reply by Kirill-code** [reply]
Hi @melohagan  I tried this https://github.com/Budibase/budibase/blob/develop/docs/DEV-SETUP-WINDOWS.md Want to make some changes and maybe create additional connector.
